Traceback (most recent call last):

  File "/app/bot.py", line 134, in <module>

    asyncio.run(main())

  File "/root/.nix-profile/lib/python3.11/asyncio/runners.py", line 190, in run

    return runner.run(main)

           ^^^^^^^^^^^^^^^^

  File "/root/.nix-profile/lib/python3.11/asyncio/runners.py", line 118, in run

    return self._loop.run_until_complete(task)